apps/antalmanac/src/providers/Themev5.tsx

```diff
@@ -24,7 +24,7 @@ const darkTheme: PaletteOptions = {
         main: '#ffffff',
     },
     background: {
-        default: '#303030',
+        default: darkModePalette.DARK_BODY_BACKGROUND,
```

Comment:
I'm concerned about this... what is a better way?

Reply:
Could be worse